pkg/webhooks/leaderworkerset_webhook_test.go

```diff
@@ -184,3 +189,100 @@ func TestIsNotMoreThan100Percent(t *testing.T) {
 		})
 	}
 }
+
+func TestGeneralValidate(t *testing.T) {
+	type args struct {
+		obj runtime.Object
+	}
+	tests := []struct {
+		name    string
+		r       *LeaderWorkerSetWebhook
+		args    args
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
HI @congcongke 
Does the TODO need to be removed?

Reply:
done